lib/services/dartservices.dart

```diff
@@ -481,20 +483,35 @@ class CandidateFix {
           .map<SourceEdit>((value) => new SourceEdit.fromJson(value))
           .toList();
     }
+    if (_json.containsKey("linkedEditGroups")) {
+      linkedEditGroups = (_json["linkedEditGroups"] as core.List)
+          .map<LinkedEditGroup>((value) => new LinkedEditGroup.fromJson(value))
+          .toList();
+    }
     if (_json.containsKey("message")) {
       message = _json["message"];
     }
+    if (_json.containsKey("selectionOffset")) {
+      selectionOffset = _json["selectionOffset"];
+    }
   }
 
   core.Map<core.String, core.Object> toJson() {
     final core.Map<core.String, core.Object> _json =
-        new core.Map<core.String, core.Object>();
+    new core.Map<core.String, core.Object>();
```

Comment:
This indent seems wrong. Rerun `dartfmt`?

Reply:
This is generated code, so I hadn't formatted it (same reason there's a `new` in there).

Just ran the formatter, though.